### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by stevepiercy**
This PR contains trivial updates after I ran `make linkcheck` and `make html`. There are no substantial content revisions.

### CODE REVIEW

**Review by thyarles — APPROVED** [review]
LGTM

File: docs/conf.py [hunk 1]
```diff
@@ -54,10 +54,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = '1.2.0'
-# The full version, including alpha/beta/rc tags.
-release = '1.2.0'
+with open('../version.txt', 'r') as reader:
```
Comment (Rudd-O) [inline docs/conf.py:57]: You may want to make that path relative to the `conf.py` file using os.path.relpath and __file__.

**Review by Rudd-O — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (stevepiercy) [reply docs/conf.py:57]: Do you mean this?

```python
with open(os.path.relpath('../version.txt'), 'r') as reader:
```

I don't know why using `os.path.relpath` would be better, but I would appreciate it if you can enlighten me.

Comment (stevepiercy) [reply docs/conf.py]: Done in [83758d3](https://github.com/plone/ansible-playbook/pull/142/commits/83758d3f07925508d2c3b5510e92f25d220175d3).

**Pull request merged**